docs/docs/data-structures/tree/schema-definition.mdx

```diff
@@ -184,3 +184,79 @@ class Proposal = sf.object('Proposal', {
     text: sf.optional(sf.string),
 });
 ```
+
+## Best Practices
```

Comment:
I moved this section out of types-of-changes (and did not modify it).
My thinking is that this is advice that is most useful before app authors write their schemas as opposed to after.